include/revng/Model/Type.h

```diff
@@ -85,6 +85,7 @@ class model::Type : public model::generated::Type {
   bool verify(bool Assert) const debug_function;
   RecursiveCoroutine<bool> verify(VerifyHelper &VH) const;
   void dump() const debug_function;
+  void dumpDotOnFile(const char *FileName) const debug_function;
```

Comment:
* @pfez was saddened by the fact that the graph doesn't allow to easily get to a function prototype.
What do you think of moving this to `model::Binary` and include a node for each `model::Function` and an edge for the prototype and another for the stack frame.
We could also do this for `model::Segment`s, once antonio's stuff is in.
* Rename the method to `dumpTypeGraph`, call the argument "Path" and use `StringRef` instead of `const char *`.

Reply:
Small aside, I believe using `const char *` is a trick for enabling to call the function from `gdb`. If that's the case we could have 2 overloads, one with `const char *` and one with `StringRef`. Does this make sense?

Reply:
@pfez right! Let's leave it like it is.

Reply:
1. yes good idea thanks for the suggestion. I also realized typedefs are missing, so I'll add those.
2. fez is right